jobs/mongodb_backup_job.go

```diff
@@ -119,6 +154,10 @@ func (j *MongoDBBackupJob) Run(ctx context.Context, send Send) error {
 		},
 	})
 
+	select {
+	case <-ctx.Done():
+	case <-time.After(waitForLogs):
```

Comment:
2.2 I would replace with that, we don't have any new logs `len(logs[skip:]) == 0. But in fact, it's the same

Reply:
Anyway, this still can't guarantee that we will receive all logs, what if in this interval we don't receive anything new.

+ this goroutine can hang more then necessary. I can see message "backup finished", but then I'll wait another period.

Reply:
Because for now we can't get logs from `pbm` by chunks and we grabbing the whole log each time, I suggest compare not the last line, but log length.

And we should ask `pbm` team for some marker that will show us that the log is finished. Probably it's possible form their side.

Reply:
Unforunately there is no market.

Anyway are not delayed. There was issue with our handling when backup finished. That's fixed. But I keep some small delay just in case.